app/aboot/lk2nd-motorola.h

```diff
@@ -0,0 +1,26 @@
+#ifndef _LK2ND_MOTOROLA
+#define _LK2ND_MOTOROLA
+
+#define MMI_UNIT_INFO_VER 3
```

Comment:
I think it would be better to use the header as-is from https://github.com/msm8916-mainline/android_kernel_qcom_msm8916/blob/harpia/drivers/soc/qcom/mmi-unit-info.h, without modifications.
That preserves the license header. You could consider removing `mmi_set_pureason` from the header, but on the other hand it shouldn't hurt if you keep it in there (unused).